### PR TITLE
daily driver

### PR DESCRIPTION
**Pull request by valeIT**

### CODE REVIEW

File: _posts/2019-12-15-phone-or-laptop.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+[The Tablet Loses](https://brooksreview.net/2014/09/the-tablet-loses/) :
+>If you could only buy one device out of a laptop, tablet, or phone which would you buy? If it is me I’m buying the phone. It would be annoying, certainly, but my iPhone can do everything my iPad can do, and my iPad can do almost everything my Mac can do. So, there’s that.
+
+I'm not on the same boat at all.
+
+If I'd have to choose between my Mac and my iPhone I'd choose my Mac, hands down. The iPhone is just way too limited. Too limited to read (books, and long articles), too limited to write on (as before novels or long articles), too limited to work on (no Xcode, no easy to use terminal, ftp, ssh, no good filesystem). Okay, I don't have an external keyboard for it and with one it might be easier to type on it, but who in their right minds would have one for an iPhone? How would you be able to read what you write from so far away? On an iPad it is different and if you're a manager, a writer or a podcaster it might be enough for you. For programmers it is not and even if Apple will one day add Xcode to it which does not seem like it is happening soon most programmers I know will still use a mac instead.
```
Comment (MsSlacker) [inline _posts/2019-12-15-phone-or-laptop.md]: minds - > mind

Comment (MsSlacker) [reply _posts/2019-12-15-phone-or-laptop.md]: manager - - what's the space there for

Comment (MsSlacker) [reply _posts/2019-12-15-phone-or-laptop.md]: which does not seem like it is happening soon - > put in brackets

Comment (MsSlacker) [reply _posts/2019-12-15-phone-or-laptop.md]: Second programmers is useless

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2019-12-15-phone-or-laptop.md (hunk 1 above)
Comment (valeIT) [reply _posts/2019-12-15-phone-or-laptop.md]: > manager - - what's the space there for

Not sure what you mean with that..?

Comment (valeIT) [reply _posts/2019-12-15-phone-or-laptop.md]: Rest is fixed

Comment (valeIT) [reply _posts/2019-12-15-phone-or-laptop.md]: > > manager - - what's the space there for
> 
> Not sure what you mean with that..?

@MsSlacker ^

File: _posts/2019-12-15-phone-or-laptop.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+[The Tablet Loses](https://brooksreview.net/2014/09/the-tablet-loses/) :
+>If you could only buy one device out of a laptop, tablet, or phone which would you buy? If it is me I’m buying the phone. It would be annoying, certainly, but my iPhone can do everything my iPad can do, and my iPad can do almost everything my Mac can do. So, there’s that.
+
+I'm not on the same boat at all.
+
+If I'd have to choose between my Mac and my iPhone I'd choose my Mac, hands down. The iPhone is just way too limited. Too limited to read (books, and long articles), too limited to write on (as before novels or long articles), too limited to work on (no Xcode, no easy to use terminal, ftp, ssh, no good filesystem). Okay, I don't have an external keyboard for it and with one it might be easier to type on it, but who in their right mind would have one for an iPhone? How would you be able to read what you write from so far away? On an iPad it is different and if you're a manager, a writer or a podcaster it might be enough for you. For programmers it is not, and even if Apple will one day add Xcode to it - which does not seem like it is happening soon  - most people will keep using a mac instead.
```
Comment (MsSlacker) [inline _posts/2019-12-15-phone-or-laptop.md]: manager

I still see space after that word and before ","

**Comment by valeIT** [comment]
@MsSlacker

**Review by MsSlacker — APPROVED** [review]

**Pull request merged**